integration_tests/two_sided_pay_as_clear_market.feature

```diff
@@ -5,6 +5,7 @@ Feature: Two sided pay_as_clear market tests
      And d3a is installed
      When we run the simulation with setup file two_sided_pay_as_clear.default_2a and parameters [24, 15, 15, 0, 1]
      Then all trades are equal to market_clearing_rate
+     And buyers and sellers are not same
```

Comment:
So you introduced the `already_tracked` flag to disable that devices buy their own energy? Did I get it right? 
Was that the problem? Does not seam to coincide with the Jira ticket description.

Reply:
It was discussed already offline, but I would like to point out that we need to use Jira more an write comments if the story context has changed or if the root cause of the error is completely different from the description of the issue. That will make it easier for everyone to keep track of the issue, especially during reviews.

Reply:
And I also suggest to use the general comment field of the PR (when opening) for a short summary what has been done.
A lot of times I realised, that I forgot to add stuff when I was writing this summary.